Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,469 @@
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={2}
+                    colStart={3}
+                    borderTopWidth='1px'
+                    borderLeftWidth='1px'
+                    borderRightWidth='1px'
+                    borderColor={borderCol}
+                >
+                    <Center direction='row' pl='1' pr='1' h='100%'>
+                        <Text fontSize='sm'>Accelerator Out</Text>
+                        <Spacer/>
+                        <Text fontSize='sm'>{props.data?.accelerator_out[0].toFixed(1) ?? -1.0} {CONSTANTS.accelerator_out.UNIT}</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={3}
+                    colStart={3}
+                    borderTopWidth='1px'
+                    borderLeftWidth='1px'
+                    borderRightWidth='1px'
+                    borderColor={borderCol}
+                >
+                    <Center direction='row' pl='1' pr='1' h='100%'>
+                        <Text fontSize='sm'>Speed</Text>
+                        <Spacer/>
+                        <Text fontSize='sm'>{props.data?.speed[0].toFixed(0) ?? -1.0} {CONSTANTS.speed.UNIT}</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={4}
+                    colStart={3}
+                    borderTopWidth='1px'
+                    borderLeftWidth='1px'
+                    borderRightWidth='1px'
+                    borderColor={borderCol}
+                    pl='1' 
+                    pr='1'
+                >
+                    <RangeCell
+                        fontSize='xs'
+                        rangeBarH='1vh'
+                        label='Motor Power'
+                        data={props.data?.motor_power[0] ?? -1.0}
+                        digits={0}
+                        unit={CONSTANTS.motor_power.UNIT}
+                        min={CONSTANTS.motor_power.MIN}
+                        max={CONSTANTS.motor_power.MAX}
+                    />
+                </GridItem>
+                <GridItem
+                    rowStart={5}
+                    colStart={3}
+                    borderWidth='1px'
+                    borderColor={borderCol}
+                    pl='1' 
+                    pr='1'
+                >
+                    <RangeCell
+                        fontSize='xs'
+                        rangeBarH='1vh'
+                        label='Motor Current'
+                        data={props.data?.motor_current[0] ?? -1.0}
+                        digits={0}
+                        unit={CONSTANTS.motor_current.UNIT}
+                        min={CONSTANTS.motor_current.MIN}
+                        max={CONSTANTS.motor_current.MAX}
+                    />
```

Comment:
```suggestion
                    <RangeCell
                        fontSize='sm'
                        rangeBarH='1vh'
                        label='Motor Current'
                        data={props.data?.motor_current[0] ?? -1.0}
                        digits={0}
                        unit={CONSTANTS.motor_current.UNIT}
                        min={CONSTANTS.motor_current.MIN}
                        max={CONSTANTS.motor_current.MAX}
                    />
```
Can you please change the fontSize to `sm`? If it doesn't fit for you though, lmk